### ISSUE TITLE
Chromecast support

### ISSUE THREAD

**Issue by nnuel** [felicianotech/pocket-casts-desktop-app#55]
Would it be possible to get a button to share to a chromecast (audio) like it would have in the web-version on Chromium?

![image](https://user-images.githubusercontent.com/4148154/152534665-380c4900-8d6c-4cdf-ad0a-a1203c70d714.png)

As the app is built using electron, it should in theory be possible I think.

https://github.com/GPMDP/electron-chromecast

**Comment by felicianotech** [comment]
That would be cool. I'll look into this.

**Comment by nnuel** [comment]
Yes, it would be great to push the podcast url to a chromecast, as it would make for easier listening during cooking/cleaning and programming too ofcourse! Not sure how the backend on the pocket-casts site works with that.
